tests/ci/run_cross_ppc_tests.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0 OR ISC
+
+SCRIPT_DIR="$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
+SCRIPT_DIR="$(readlink -f "${SCRIPT_DIR}")"
+
+"${SCRIPT_DIR}/run_cross_tests.sh" ppc powerpc-unknown-linux-gnu "-DCMAKE_BUILD_TYPE=Release" "-DCMAKE_BUILD_TYPE=Release -DFIPS=1 -DBUILD_SHARED_LIBS=1"
```

Comment:
`ppc` (without qualifier) is implicitly 32-bit?

Reply:
also, why do we build this as FIPS but ppc64 as non-fips?

Reply:
* Yeah, "ppc" is the right value to use. (One resource I use for checking such values is [cross-rs](https://github.com/cross-rs/cross/blob/main/docker/Dockerfile.powerpc-unknown-linux-gnu#L48C5-L48C33). Their [CMake toolchain file](https://github.com/cross-rs/cross/blob/main/docker/toolchain.cmake) has most of the important variable.)
* We're still researching a problem with the FIPS build on ppc64.